src/cli/sync.ts

```diff
@@ -151,6 +151,10 @@ export async function handler(argv: Arguments) {
       log.warning(`Source video file does not exist: ${err.path}`);
       return;
     }
+    if(err.message.includes("MUX_TOKEN_ID environment variable is missing or empty") || err.message.includes("MUX_TOKEN_SECRET environment variable is missing or empty")){
+      log.error(`${err.message} Visit \x1b[4;34mhttps://next-video.dev/docs#remote-storage-and-optimization\x1b[0m for more information.`);
```

Comment:
Instead of logging the error message, let's just say

> Mux MUX_TOKEN_ID or MUX_TOKEN_SECRET can't be found. 

And then the "Visit [link]" after that.
These users won't be using the Mux SDK directly, so the added info about what to do with the Mux SDK would be more confusing than helpful.

Reply:
Otherwise lgtm 👍